CollectionCell.xctemplate/___VARIABLE_sourceCellIdentifier___CollectionCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ___FILENAME___
+//  ___PROJECTNAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___ using Collection View Cellkit Template (v1.0).
```

Comment:
Also, CellKit has large K. 🙂

```suggestion
//  Created by ___FULLUSERNAME___ on ___DATE___ using CellKit Collection View Cell Template (v1.0).
```